sound/soc/sof/topology.c

```diff
@@ -1819,6 +1774,17 @@ static int sof_process_load(struct snd_soc_component *scomp, int index,
 
 	sof_dbg_comp_config(scomp, &process->config);
 
+	process->size = ipc_data_size;
+	swidget->private = process;
+
+	ret = sof_ipc_tx_message(sdev->ipc, process->comp.hdr.cmd, process,
+				 ipc_size, r, sizeof(*r));
+
+	if (ret < 0) {
+		dev_err(sdev->dev, "error: failed to create process comp\n");
+		goto err;
+	}
+
 	/*
 	 * found private data in control, so copy it.
 	 * get possible component controls - get size of all pdata,
```

Comment:
@juimonen the comment is no longer valid.

Reply:
ok